.gitignore

```diff
@@ -40,3 +40,5 @@ temp.jl
 yeast_test.xml
 ecoli_test.xml
 ecoli_test.json
+test.html
+test.out
```

Comment:
EOL missing

Reply:
(btw gitignore should generally ignore only stuff that _needs_ to be ignored by all deployments, not stuff that you randomly placed in the repo and don't want to push it again. Others might have different usecases for that.)